### PR TITLE
Redcap dal

### PR DESCRIPTION
**Pull request by kairstenfay**
Remove custom POST requests to the REDCap API in favor of using the ID3C REDCap DAL.

Depends on seattleflu/id3c#202

### CODE REVIEW

**Comment by kairstenfay** [comment]
[Move custom POSTs to REDCap API in random backoffice Python scripts to the id3c data access layer](https://trello.com/c/G8HPR9mZ/961-move-custom-posts-to-redcap-api-in-random-backoffice-python-scripts-to-the-id3c-data-access-layer)

File: bin/redcap-data-dictionary/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# REDCap data dictionary manipulations
+
+Use the scripts in this directory for programmatically manipulating a project's REDCap data dictionary to include specified HTML attributes.
+
+This workflow is used for injecting `<span lang="{ISO-code}"></span>` HTML attributes into our data dictionaries in translations of the SCAN REDCap project.
```
Comment (jccraft-uw) [inline bin/redcap-data-dictionary/README.md:5]: Could you please add why we want to inject the lang attribute into the data dictionaries? What does that <span lang ...> get used for/by?

**Review by jccraft-uw — APPROVED** [review]

File: bin/redcap-data-dictionary/README.md (hunk 1 above)
Comment (kairstenfay) [reply bin/redcap-data-dictionary/README.md:5]: Updated in the latest push.

Comment (jccraft-uw) [reply bin/redcap-data-dictionary/README.md:5]: Thank you!

**Pull request merged**